### ISSUE TITLE
Privacy Badger support

### ISSUE THREAD

**Issue by nolanlawson** [nolanlawson/pinafore#23]
Privacy Badger thinks that Pinafore is running "third-party trackers" because of CORS requests to instances. Sigh...

Backstory: https://mstdn.io/@lychee/99830242316872578

**Comment by nolanlawson** [comment]
Apparently there is a workaround if I host a `.well-known/dnt-policy.txt` file on the server: https://mastodon.xyz/@xor/99830613210517556

**Comment by nolanlawson** [comment]
Hmmm unfortunately I may not actually be able to abide by the rules of the `dnt-policy.txt` because it says:

```
Logs with DNT Users' identifiers removed (but including IP addresses and
User Agent strings) may be retained for a period of 10 days or less,
unless an Exception (below) applies.
```

And I'm currently using now.sh, which [stores 1GB of server logs by default](https://zeit.co/account/plan) on the Premium plan I'm using, and it doesn't seem possible to reduce that.

Maybe for now it's best if folks just disable Privacy Badger on Pinafore.

**Comment by nolanlawson** [comment]
I just tested Privacy Badger on Firefox Dev Edition 60 with a variety of instances and couldn't repro the issue where users are unable to add an instance due to PB blocking. For all instances I tested (mastodon.social, toot.cafe, malfunctioning.technology, freedom.horse) it called some of them "potential trackers" but didn't block them by default.

Since this appears to be something that Pinafore itself cannot solve (the instances can solve it by adding the `dnt-policy.txt` file, and since users can configure PB to work correctly, it seems best just to close this issue for now.
